### PR TITLE
Improve session survey screen

### PR DESCRIPTION
**Pull request by futabooo**
## Issue
- #65

## Overview (Required)
- Add SurveyItem Enum Class
- Improve session survey screen design

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/survey/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    <string name="total_evaluation">セッションはどうでしたか?(総合評価)</string>
-    <string name="relevancy">セッションはあなたの仕事やプロジェクトに関連がありましたか?</string>
-    <string name="as_expected">セッションの説明を読んで期待した通りの内容でしたか?</string>
-    <string name="difficulty">セッションの難易度はいかがでしたか?</string>
-    <string name="knowledgeable">技術的な知見は得られたでしょうか?</string>
-    <string name="comment">他にコメントがあればぜひ教えてください</string>
-    <string name="comment_hint">個人が特定できるような内容は書かないでください</string>
-    <string name="submit">送信する</string>
+    <string name="total_evaluation">How was the session?\n(Overall Rating)</string>
+    <string name="relevancy">How relevant was this session to your projects?</string>
```
Comment (panpanini) [inline feature/survey/src/main/res/values/strings.xml]: `How relevant was this session to your work or personal projects?`

File: feature/survey/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    <string name="as_expected">Based on the description, was the content as expected?</string>
+    <string name="difficulty">Was this session easy to understand?</string>
+    <string name="knowledgeable">How knowledgeable was the content?</string>
```
Comment (panpanini) [inline feature/survey/src/main/res/values/strings.xml]: `Were you able to gain useful knowledge from the content?`

**Comment by futabooo** [comment]
@panpanini Thank you for your review! And I fixed it

File: feature/survey/src/main/java/io/github/droidkaigi/confsched2019/survey/ui/SessionSurveyFragment.kt [hunk 3]
```diff
@@ -203,23 +202,40 @@ class SessionSurveyFragment : DaggerFragment() {
...
+    }
+}
 
-            override fun onPageScrolled(
-                position: Int,
-                positionOffset: Float,
-                positionOffsetPixels: Int
-            ) {
-            }
+private fun EditText.onTextChange(onChanged: (text: String) -> Unit) {
```
Comment (takahirom) [inline feature/survey/src/main/java/io/github/droidkaigi/confsched2019/survey/ui/SessionSurveyFragment.kt]: 👍

File: feature/survey/src/main/res/layout/item_session_survey.xml [hunk 4]
```diff
@@ -1,39 +1,62 @@
...
+            android:layout_marginEnd="16dp"
+            android:layout_marginStart="16dp"
             android:gravity="center_vertical"
             android:hint="@string/comment_hint"
-            android:paddingStart="10dp"
             android:paddingEnd="10dp"
-            android:visibility="gone"
-            tools:visibility="visible" />
+            android:paddingStart="10dp"
+            android:visibility="@{surveyItem.isComment ? View.VISIBLE : View.GONE}"
```
[the remark this answers is not in the document]
Comment (futabooo) [reply feature/survey/src/main/res/layout/item_session_survey.xml]: ViewBinding.kt is in session package, so if i use `app:visibleGone=` i create ViewBinding.kt in survey package?
or export ViewBinding.kt to another package and import to session package and survey package?

File: feature/survey/src/main/java/io/github/droidkaigi/confsched2019/survey/ui/SessionSurveyFragment.kt [hunk 5]
```diff
@@ -200,26 +199,43 @@ class SessionSurveyFragment : DaggerFragment() {
 
             override fun isViewFromObject(view: View, `object`: Any): Boolean = view == `object`
 
-            override fun getCount(): Int = 6
+             override fun getCount(): Int = enumValues<SurveyItem>().size
```
[the remark this answers is not in the document]
Comment (takahirom) [reply feature/survey/src/main/java/io/github/droidkaigi/confsched2019/survey/ui/SessionSurveyFragment.kt]: 🙇 🙇 🙇 

```suggestion
            override fun getCount(): Int = enumValues<SurveyItem>().size
```

File: feature/survey/src/main/res/layout/item_session_survey.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (takahirom) [reply feature/survey/src/main/res/layout/item_session_survey.xml]: I tried import ViewBinding.kt but I can not do it.
Can you copy ViewBinding.kt to survey module?

[the remark this answers is not in the document]
Comment (futabooo) [reply feature/survey/src/main/res/layout/item_session_survey.xml]: ok! I will try it!

**Comment by futabooo** [comment]
@takahirom Thank you for your review! fix review!

**Pull request merged**